components/general/InfoBlock/InfoBlock.tsx

```diff
@@ -56,11 +56,15 @@ const InfoBlock = ({
             {content}
           </p>
           <form className={styles.info_block__sign_up_content__form}>
-            <input
-              type="email"
-              defaultValue="YOUR EMAIL"
-              className={`global__heading-h4 ${fontSyne.className} ${styles.info_block__sign_up_content__form__input}`}
-            />
+            <div className={styles.info_block__sign_up_content__form__group}>
+              <label htmlFor="email" className={`global__heading-h4 ${fontSyne.className}`}>
+                YOUR EMAIL
+              </label>
```

Comment:
One possible solution for keep label exist and hide it is put it outside the user view like follows:
<img width="268" alt="image" src="https://user-images.githubusercontent.com/70834897/222638574-bc3ce703-5d64-4bc1-9a01-93c1e85e59ea.png">